packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.0
+
+* Adds federated Windows support.
```

Comment:
I would just say "Adds Windows support." ("federated" might make some people think it's not endorsed yet, since the terms are still fairly new in the ecosystem.)

Reply:
Done.